.github/workflows/unit-tests.yaml

```
@@ -17,7 +17,8 @@ jobs:
       fail-fast: false
       matrix:
         gotags: [ 'GOTAGS=""', 'GOTAGS=release' ]
-    runs-on: ubuntu-latest
+        arch: [amd64, arm64]
+    runs-on: ${{ (matrix.arch == 'arm64' && 'ubuntu-24.04-arm') || 'ubuntu-24.04' }}
```

Would this work:
```suggestion
    runs-on: ubuntu-24.04${{ (matrix.arch == 'arm64' && '-arm') || '' }}
```
and make it clearer (?) it's just about the suffix?